guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -266,6 +266,14 @@ If the script fails due to missing or outdated packages, you must download and i
 # hash -d {foreman-maintain} service 2> /dev/null
 ----
 
+. If you have migrated content from Pulp 2 to Pulp 3, remove all Pulp 2 content.
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} content remove-pulp2
+----
+This removes uploaded and synchronized Pulp 2 packages, content in `/var/lib/pulp/content/`, the mongo database, and migration content in the Pulp 3 database.
```

Comment:
```suggestion
This removes uploaded and synchronized Pulp 2 packages, content in `/var/lib/pulp/content/`, MongoDB, and migration content in the Pulp 3 database.
```

We must follow spelling and capitalization conventions for a third party product as per their branding.